test/test_files/tinysnb/function/list.test

```diff
@@ -72,12 +72,18 @@
 False
 False
 True
+-STATEMENT RETURN [1, 2.0];
+---- 1
+[1.000000,2.000000]
+-STATEMENT RETURN [null, 3.0, 3];
+---- 1
+[,3.000000,3.000000]
 -STATEMENT RETURN [1, 'a'];
```

Comment:
Duckdb will convert 1 to string in such case:
```
D select [1, 'a'];
┌─────────────────────────┐
│ main.list_value(1, 'a') │
│        varchar[]        │
├─────────────────────────┤
│ [1, a]                  │
└─────────────────────────┘
```

Reply:
I'm pretty sure this is no longer the case in new version of Duckdb